lib/stylesheet/importer.rb

```diff
@@ -43,7 +43,7 @@ def self.register_import(name, &blk)
       theme&.all_theme_variables&.each do |field|
         if field.type_id == ThemeField.types[:theme_upload_var]
           if upload = field.upload
-            url = upload_cdn_path(upload.url)
+            url = upload.url
```

Comment:
Why are we removing the `cdn`? Was it unnecessary?

Reply:
It looks like `Upload.url` is the full CDN URL when a CDN is used or path to local file otherwise.

I do not see any point in using `upload_cdn_path`. I see it is only used in two more places, in `session_controller` for SSO card and profile background. I also have a feeling that it is not required in those places either, but I did not get around to see how it works.